src/components/Footer.js

```diff
@@ -70,6 +71,9 @@ const Footer = () => (
           <a href="https://twitter.com/_astrocoders" target="_blank" rel="noopener noreferrer">
             <Twitter />
           </a>
+          <a href="https://medium.com/astrocoders" target="_blank" rel="noopener noreferrer">
+            <Medium />
+          </a>
         </SocialLinks>
         <Copyrights>Copyright 2018 © Astrocoders Desenvolvimento LTDA</Copyrights>
```

Comment:
We got a feedback to write this here in English as well. 
So maybe just `Copyright {new Date().getFullYear()} © Astrocoders`